BlueprintUI/Sources/Layout/Empty.swift

```diff
@@ -16,7 +16,7 @@ public struct Empty : Element {
     public init() {}
     
     public var content: ElementContent {
-        ElementContent { _ in
+        .withMeasurement { _ in
```

Comment:
Calls that aren't using the constraint could be replaced with `ElementContent(intrinsicSize:)` to dodge this entirely.

Reply:
Changed